agave_app/agaveGui.cpp

```diff
@@ -1216,3 +1256,25 @@ agaveGui::appToViewerState()
 
   return v;
 }
+
+void
+agaveGui::changeEvent(QEvent* event)
+{
+  if (event->type() == QEvent::ThemeChange) {
+    // check for dark or light mode
+    auto sh = QGuiApplication::styleHints();
+    auto colorScheme = sh->colorScheme();
+    if (m_colorScheme == colorScheme) {
+      return;
+    }
+    if (colorScheme == Qt::ColorScheme::Dark) {
+      setStyleSheet(darkStyleSheet);
+      LOG_DEBUG << "ThemeChange to Dark";
+    } else if (colorScheme == Qt::ColorScheme::Light) {
+      setStyleSheet(lightStyleSheet);
+      LOG_DEBUG << "ThemeChange to Light";
+    }

```

Comment:
just curious: why is this code more or less a repeat of what's above? in lines 80-84. is there more than one way the theme gets changed?

Reply:
lines 80-84 are a one-time initialization.  This code here handles dynamic changes.  If I had to repeat the logic one more time then I'd refactor, or if the logic was any more complicated than this.  Maybe I'll change it so that it reads something like setStyleSheet(getStyleSheetString(colorScheme))